### PR TITLE
Remove JavaScript SDK injection

### PR DESCRIPTION
**Pull request by jrabbott**
### Context
Support click analysis in App Insights

### Change proposed in this pull request
JavaScript injection provides a default configuration experience. Required to remove auto-injection and manually add the JavaScript SDK.

### Guidance to review 
https://learn.microsoft.com/en-us/azure/azure-monitor/app/asp-net-core?tabs=netcorenew%2Cnetcore6#enable-client-side-telemetry-for-web-applications

https://learn.microsoft.com/en-gb/azure/azure-monitor/app/javascript-feature-extensions?tabs=javascriptwebsdkloaderscript

### Checklist
- [x] ~~Work items have been linked [(use AB#)](https://learn.microsoft.com/en-us/azure/devops/boards/github/link-to-from-github?view=azure-devops#use-ab-to-link-from-github-to-azure-boards-work-items)~~
- [x] Your code builds clean without any errors or warnings
- [x] You have run all unit/integration tests and they pass
- [x] Your branch has been rebased onto main
- [x] You have tested by running locally

### CODE REVIEW

File: web/src/EducationBenchmarking.Web/Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -22,8 +26,34 @@
     <link rel="manifest" href="/assets/manifest.json">
     <link rel="stylesheet" href="/css/front-end.css"/>
     <link rel="stylesheet" href="/css/main.css" asp-append-version="true"/>
+    @if (!string.IsNullOrEmpty(connString))
+    {
+        <script type="text/javascript" src="https://js.monitor.azure.com/scripts/b/ext/ai.clck.2.min.js"></script>
+        <script type="text/javascript">
+ 		    var clickPluginInstance = new Microsoft.ApplicationInsights.ClickAnalyticsPlugin();
```
Comment (WolfyUK) [inline web/src/EducationBenchmarking.Web/Views/Shared/_Layout.cshtml:33]: [nitpick] Tab spacing looks a little off

File: web/src/EducationBenchmarking.Web/Views/Shared/_Layout.cshtml
Comment (WolfyUK) [inline web/src/EducationBenchmarking.Web/Views/Shared/_Layout.cshtml:1]: Looks good - have you seen what this produces in App Insights?

File: web/src/EducationBenchmarking.Web/Views/Shared/_Layout.cshtml (hunk 1 above)
Comment (jrabbott) [reply web/src/EducationBenchmarking.Web/Views/Shared/_Layout.cshtml:33]: Will have a look. Annoyingly Rider refuses to reformat it.

File: web/src/EducationBenchmarking.Web/Views/Shared/_Layout.cshtml
Comment (jrabbott) [reply web/src/EducationBenchmarking.Web/Views/Shared/_Layout.cshtml:1]: Not yet - as I didn't have the connection string locally.

Comment (jrabbott) [reply web/src/EducationBenchmarking.Web/Views/Shared/_Layout.cshtml:1]: image

Comment (jrabbott) [reply web/src/EducationBenchmarking.Web/Views/Shared/_Layout.cshtml:1]: Displaying as expected - custom event with name as the label.

**Review by WolfyUK — APPROVED** [review]

**Pull request merged**